### ISSUE TITLE
Please add Android Wear Complication

### ISSUE THREAD

**Issue by ins5736** [MilosKozak/AndroidAPS#728]
This is a request for an enhancement. I'd like to be able to read BG data in a complication for Android Wear rather than using a dedicated watch face.

**Comment by jotomo** [comment]
AAPS will need to implement Wear 2 support in order to support complications. If you only need a current SGV and a trend and are using xDrip+, you can use the complication from xDrip+ for now.

**Comment by ins5736** [comment]
Oddly enough, I already tried xdrip+. I was using it in sync/follower mode to local broadcast from NS Client. The Xdrip+ complication only displays "null, null" instead of the bg.

**Comment by jotomo** [comment]
Did you enable the Android Wear integration in xDrip?

**Comment by gregorybel** [comment]
it seems xdrip complication doesn't work in follower mode but it will not help you to have it into aaps if you are still the follower, or do I miss something?

**Comment by ins5736** [comment]
Yes, I enabled the Xdrip+ Android wear integration. The watchfaces for Xdrip+ worked fine. The complication did not.

If AAPS supported Android Wear 2, then I wouldn't need to follow it from Xdrip+, I could just use AAPS alone.

**Comment by gregorybel** [comment]
hum, still don't understand your setup, could you please describe what is with what connected?

if we can use aaps alone, why not xdrip too?

**Comment by ins5736** [comment]
The xdrip+/NS Client set is described here: https://jamorham.gitbooks.io/xdrip-plus/content/xdrip-plus-nightscout-viewer.html

I don't understand your second question.

**Comment by gregorybel** [comment]
Can you explain Which CGM you use, Which pump, is pump connected to phone or a rig? do you use 2 phones? etc...

on my side, quiet easy: one phone connected to CGM, pump and watch. no needs for NS

**Comment by ins5736** [comment]
My wife uses a dexcom g5, collected with xdrip+, uploaded to Nighstscout, and NightScout is read by my phone.

We recently purchased TicWatch E's as Pebble's end of service is in June and wanted to shift watch platforms ahead of that.

Mainly, I'd like to be able to see my wife's bg in a complication since it'd free up my choices of watchface.

**Comment by Arakon** [comment]
I'll attach myself to this feature request and second it.. it'd be awesome to use any "pretty" watchface and still have my Dexcom data. I'm required to actually use the original dexcom app by my insurance and doctor, and running xdrip in parallel causes problems sometimes (lost data).

**Comment by jotomo** [comment]
@ins5736 You may try a recent xDrip version, there's a fix mid-May which may fix your issue using xDrip complication in follower mode.

**Comment by ins5736** [comment]
Thank you, I already know about that. Unfortunately it didn't resolve the issue for me for another issue user.

**Comment by ins5736** [comment]
Realized I never followed up on issue. Currently on 1/10/19 xdrip and complication still reads null, null.

**Comment by dlvoy** [comment]
I currently working on it

**Comment by dlvoy** [comment]
![complications](https://user-images.githubusercontent.com/5094588/68943268-000ee680-07ab-11ea-8d2f-85cbf85134f3.jpg)
![complications_on_watchfaces](https://user-images.githubusercontent.com/5094588/68943283-04d39a80-07ab-11ea-8a70-ab0a61ecb8dc.jpg)
